src/components/cards/ProfileInfo.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <v-card>
+    <v-row class="pa-4" justify="space-between">
+      <v-col cols="12">
+        <b>Name:</b> {{ name }}<br />
+        <b>Title:</b> {{ title }}<br />
+        <b>Maintainer:</b> {{ maintainer }}<br />
+        <b>Copyright:</b> {{ copyright }}<br />
+        <b>Copyright Email:</b> {{ copyright_email }}<br />
+        <b>Summary:</b> {{ summary }}<br />
+        <b>Supports:</b>
+        <ul v-for="item in supports" v-bind:key="item.name">
+          <li>
+            <b>{{ item.name }}:</b> {{ item.value }}
+          </li>
+        </ul>
+        <br />
+        <b>Version:</b> {{ version }}<br />
+        <b>SHA256:</b> {{ sha256 }}<br />
```

Comment:
If the other changes suggested are implemented then these will become `{{ profile.profile.name }}`, `{{ profile.profile.title }}`, etc, unless some sort of helper is implemented (which would basically just call `profile.profile.`).